web/ugahacks-6/next.config.js

```diff
@@ -1,12 +1,23 @@
 module.exports = {
   reactStrictMode: true,
-  async redirects(){
-    return[
+  env: {
+    HOSTNAME: "localhost",
```

Comment:
I have a feeling this hardcoding is going to break in the pipelines and deployment.

Reply:
The environment variable HOSTNAME was a temporary fix to window element not found error located in serviceWorker.js. I can remove the environment variable but the window element not found error still needs to be addressed.